docs/ref/models/querysets.txt

```diff
@@ -2445,6 +2432,35 @@ more overall work (one query for the existence check plus an extra one to later
 retrieve the results) than using ``bool(some_queryset)``, which retrieves the
 results and then checks if any were returned.
 
+``contains()``
+~~~~~~~~~~~~~~
+
+.. method:: contains(obj)
+
+Returns ``True`` if the :class:`.QuerySet` contains obj, and ``False``
+if not. This tries to perform the query in the simplest and fastest way
+possible, but it *does* execute nearly the same query as a normal
+:class:`.QuerySet` query.
```

Comment:
I find this wording it a bit confusing, I'm not quite sure what it's trying to convey (but it is late as I'm reading this).

Reply:
Tried to stick to what exists() documentation says. It's possible that's not as relevant in this context as I first thought. I guess this method adds some to the query anyway. Should I just delete this paragraph, maybe?

Reply:
IMHO, I would maybe cut it from the "but" onward.